src/components/breadcrumbs/_macro-options.md

```diff
@@ -1,9 +1,10 @@
-| Name      | Type                     | Required | Description                                                             |
-| --------- | ------------------------ | -------- | ----------------------------------------------------------------------- |
-| classes   | string                   | false    | Custom classes to add to the breadcrumbs `nav` element                  |
-| ariaLabel | string                   | false    | The `aria-label` added to the `nav` element. Defaults to “Breadcrumbs”. |
-| id        | string                   | false    | The `id` added to the `nav` element                                     |
-| itemsList | `Array<BreadcrumbsItem>` | true     | An array of [breadcrumb items](#breadcrumbsitem)                        |
+| Name      | Type                     | Required | Description                                                                                                                         |
+| --------- | ------------------------ | -------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| classes   | string                   | false    | Custom classes to add to the breadcrumbs `nav` element                                                                              |
+| ariaLabel | string                   | false    | The `aria-label` added to the `nav` element. Defaults to “Breadcrumbs”.                                                             |
+| id        | string                   | false    | The `id` added to the `nav` element                                                                                                 |
+| itemsList | `Array<BreadcrumbsItem>` | true     | An array of [breadcrumb items](#breadcrumbsitem)                                                                                    |
+| variant   | string                   | false    | Set variant to "hero-grey", "hero-dark", "hero-default", "hero-pale-blue", "hero-navy-blue" for matching color of the hero variants |
```

Comment:
`hero-default` and `hero-pale-blue` seem to style the breadcrumbs in exactly the same way. Do we need both?